rolling/distribution.yaml

```diff
@@ -2699,6 +2699,17 @@ repositories:
       url: https://github.com/iRobotEducation/irobot_create_msgs.git
       version: rolling
     status: developed
+  jacro:
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/JafarAbdi/jacro-release.git
```

Comment:
Do you want to consider housing the release repo in the ros2-gbp org instead? The advantage there is that your package will automatically be available in the next ROS 2 distribution on release day.
https://docs.ros.org/en/rolling/How-To-Guides/Releasing/First-Time-Release.html#create-a-new-release-repository

Reply:
Actually, it is more than an advantage; for `rolling`, we require that the release repository be in the `ros2-gbp` organization.